src/Units/DoRUnits.java

```diff
@@ -135,24 +135,11 @@ public DoRUnitModel(String pName, DoRUnitEnum pType, UnitRoleEnum pRole, Chassis
     @Override
     public UnitModel clone()
     {
-      // Make a copy of your weapon types.
-      if( weapons != null )
-      {
-        weapons = new ArrayList<WeaponModel>();
-        for( WeaponModel weapon : weapons )
-        {
-          weapons.add(weapon.clone());
-        }
-      }
-
       // Create a new model with the given attributes.
-      DoRUnitModel newModel = new DoRUnitModel(name, type, role, chassis, getCost(), maxAmmo, maxFuel, idleFuelBurn, visionRange, movePower,
+      DoRUnitModel newModel = new DoRUnitModel(name, type, role, chassis, moneyCost, maxAmmo, maxFuel, idleFuelBurn, visionRange, movePower,
```

Comment:
Why change `getCost()` to `moneyCost` here but not in `AWBWUnits.java`?

Reply:
Because I'm a fool.